update.sh

```diff
@@ -32,12 +32,15 @@ rm -rf ~/rhino-config
 
 # If the user has selected the option to install the mainline kernel, install it onto the system.
 if [[ -f "$HOME/.rhino/config/mainline" ]]; then
-  cd ~/rhinoupdate/kernel/
-  wget -q --show-progress --progress=bar:force https://kernel.ubuntu.com/~kernel-ppa/mainline/v5.17.5/amd64/linux-headers-5.17.5-051705-generic_5.17.5-051705.202204271406_amd64.deb
-  wget -q --show-progress --progress=bar:force https://kernel.ubuntu.com/~kernel-ppa/mainline/v5.17.5/amd64/linux-headers-5.17.5-051705_5.17.5-051705.202204271406_all.deb
-  wget -q --show-progress --progress=bar:force https://kernel.ubuntu.com/~kernel-ppa/mainline/v5.17.5/amd64/linux-image-unsigned-5.17.5-051705-generic_5.17.5-051705.202204271406_amd64.deb
-  wget -q --show-progress --progress=bar:force https://kernel.ubuntu.com/~kernel-ppa/mainline/v5.17.5/amd64/linux-modules-5.17.5-051705-generic_5.17.5-051705.202204271406_amd64.deb
-  sudo apt install ./*.deb
+  if [[ ! -f "$HOME/.rhino/config/5-17-5" ]]; then
```

Comment:
Wouldn't it be better to check if the packages are already installed on the system by querying APT, instead of creating new files?

Reply:
I just believed this would be the most simple solution. Furthermore as Ubuntu devel upgrades their repositories there may be times where the Ubuntu kernel is installed over the mainline one, with matching version numbers. Wouldn't this create more issues in the long run?